Start-C4BNexusSteup.ps1

```diff
@@ -981,7 +981,7 @@ choco apikey -s 'ChocolateyInternal' -k $NugetApiKey
 # Install a non-IE browser for browsing the Nexus web portal.
 # Edge sometimes fails install due to latest Windows Updates not being installed.
 # In that scenario, Google Chrome is installed instead.
-$null = choco install microsoft-edge -y
+$null = choco install microsoft-edge -y --source="'https://chocolatey.org/api/v2/'"
```

Comment:
Same as above

Reply:
Good point! I'll address this in the next commit.